pingdom/api_responses.go

```diff
@@ -46,6 +46,21 @@ type CheckResponseType struct {
 	HTTP *CheckResponseHTTPDetails `json:"http,omitempty"`
 }
 
+type ContactResponse struct {
+	ID                 int    `json:"id"`
+	Name               string `json:"name"`
+	Email              string `json:"email,omitempty"`
+	Cellphone          string `json:"cellphone,omitempty"`
+	Countryiso         string `json:"countryiso,omitempty"`
+	Defaultsmsprovider string `json:"defaultsmsprovider,omitempty"`
+	Directtwitter      bool   `json:"directtwitter,omitempty"`
+	Twitteruser        string `json:"twitteruser,omitempty"`
+	Iphonetokens       string `json:"iphonetokens,omitempty"`
+	Androidtokens      string `json:"androidtokens,omitempty"`
```

Comment:
Would you mind capitalizing these as well to be consistent with capitalization in types?